### DISCUSSION TITLE
Graph View

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#769] [Feature Requests]
Obsidian and Foam both allow you to see all your Notes as a graph. This is super useful as your personal knowledge base grows. We should expose that in GitJournal as well.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by vHanda** [comment]
Mini Update: https://twitter.com/GitJournal_io/status/1310353515588603905

This can tried out with version `1.71` by going to `Settings -> Experimental Features`.

**Comment by alensiljak** [comment]
https://www.orgroam.com/

https://www.orgroam.com/manual.html#Graphing
